tests/conjur_variable/test.sh

```diff
@@ -26,7 +26,8 @@ function run_test_case {
     fi
 
     # You can add -vvvvv here for debugging
-    ansible-playbook 'test_cases/${test_case}/playbook.yml'
+    export SAMPLE_KEY='set_in_env'
+    ansible-playbook --extra-vars 'sample_key=set_in_extravars' 'test_cases/${test_case}/playbook.yml'
```

Comment:
This is not enough testing for this addition, we should include another test cases that uses ONLY `--extra-vars` to configure the lookup plugin.